apps/ledger-live-desktop/src/renderer/families/evm/SendAmountFields/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { Transaction as EvmTransaction } from "@ledgerhq/coin-evm/types";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { useGasOptions } from "@ledgerhq/live-common/families/evm/react";
+import { log } from "@ledgerhq/logs";
+import { AccountBridge } from "@ledgerhq/types-live";
+import React, { useCallback, useEffect, useState } from "react";
+import SendFeeMode from "~/renderer/components/SendFeeMode";
+import { EvmFamily } from "../types";
+import GasLimitField from "./GasLimitField";
+import GasPriceField from "./GasPriceField";
+import MaxFeeField from "./MaxFeeField";
+import PriorityFeeField from "./PriorityFeeField";
+import SelectFeeStrategy from "./SelectFeeStrategy";
+
+const Root: NonNullable<EvmFamily["sendAmountFields"]>["component"] = props => {
+  const { account, updateTransaction, transaction } = props;
+  const bridge: AccountBridge<EvmTransaction> = getAccountBridge(account);
+
+  const [gasOptions, error] = useGasOptions({
+    currency: account.currency,
+    transaction,
+    interval: account.currency.blockAvgTime ? account.currency.blockAvgTime * 1000 : undefined,
+  });
+
+  if (error) {
+    log("error", error.message);
```

Comment:
```suggestion
    log("SendAmountFields/index.tsx", error.message);
```

Could be worth being more specific to help with future debugging